angrmanagement/config/config_manager.py

```diff
@@ -1,7 +1,31 @@
 
-from PySide2.QtGui import QFont, QFontMetricsF
+from PySide2.QtGui import QFont, QFontMetricsF, QColor
 
 from .config_entry import ConfigurationEntry as CE
+import toml
+import logging
+
+_l = logging.getLogger(__name__)
+
+def color_constructor(config_option, value):
+    if type(value) is str:
```

Comment:
If this line is not hot, I would prefer `isinstance()` instead.

Reply:
Also `isinstance(value, basestring)` would cover `unicode` as well.